src/Sylius/Behat/Context/Api/Shop/CheckoutContext.php

```diff
@@ -52,6 +53,9 @@ final class CheckoutContext implements Context
     /** @var ApiClientInterface */
     private $countriesClient;
 
+    /** @var ApiClientInterface */
+    private $addressClient;
```

Comment:
```suggestion
    private $addressesClient;
```
to be consistent with other clients in this service